app/models/user.rb

```diff
@@ -2,4 +2,11 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'user_id'
   has_many :comments, foreign_key: 'user_id'
+  validates :name, presence: true, length: { minimum: 3, maximum: 20 }
+  validates :posts_count, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Just as the previous reviewer mentioned, Kindly ensure that you add validation tests for the `posts_count` being greater than or equal to 0. :+1: 
- [ ] Kindly ensure that your test checks that
1. When the `post counter` has a negative value, it is invalid :100: 
2. If the `post counter` is equal to zero or has a positive value, then it's valid :100: